core/query/sql.ml

```diff
@@ -239,49 +249,111 @@ and pr_base quote one_table ppf b =
   in
   let pr_b_one_table = pr_base quote one_table in
   let pr_q_true = pr_query quote true in
-  let unary_ops = StringSet.of_list ["intToString"; "stringToInt"; "intToFloat"; "floatToString"; "stringToFloat"; "floatToInt"; "not"; "negate"; "negatef"] in
-  let binary_ops = StringSet.of_list ["&&"; "||"; "=="; "<>"; "<"; ">"; "<="; ">="; "RLIKE"; "LIKE"] in
+  let unary_ops =
+    StringSet.of_list ["intToString"; "stringToInt"; "intToFloat";
+                       "floatToString"; "stringToFloat"; "floatToInt";
+                       "not"; "negate"; "negatef"] in
+  let binary_ops =
+    StringSet.of_list ["&&"; "||"; "=="; "<>"; "<"; ">"; "<="; ">="; "RLIKE"; "LIKE"] in
   let binary_map op =
     match op with
       | "&&" -> "and"
       | "||" -> "or"
       | "==" -> "="
-      | _ -> op
+      | _    -> op
   in
   let unary_map op =
     match op with
-      | "floatToInt" -> "floor"
-      | "not" -> "not "
-      | "negate" | "negatef" -> "-"
-      | "intToString" | "stringToInt" | "intToFloat" | "floatToString" | "stringToFloat" -> ""
-      | _ -> assert false
+      | "floatToInt"    -> "floor"
+      | "not"           -> "not "
+      | "negate"
+      | "negatef"       -> "-"
+      | "intToString"
+      | "stringToInt"
+      | "intToFloat"
+      | "floatToString"
+      | "stringToFloat" -> ""
+      | _               -> assert false
   in
   let pp_sql_arithmetic ppf (l, op, r) =
     match op with
-      | "/" -> Format.fprintf ppf "floor(%a/%a)" pr_b_one_table l pr_b_one_table r
-      | "^" -> Format.fprintf ppf "floor(pow(%a,%a))" pr_b_one_table l pr_b_one_table r
-      | "^." -> Format.fprintf ppf "pow(%a,%a)" pr_b_one_table l pr_b_one_table r
-      | _ -> Format.fprintf ppf "(%a%s%a)" pr_b_one_table l (Arithmetic.sql_name op) pr_b_one_table r
+    | "/" -> Format.fprintf ppf "floor(%a/%a)"
```

Comment:
Apparently because my ocaml-mode likes the bars to be aligned with the "m" in "match" rather than the "t".  However we seem to do it the latter way everywhere else... looks like I need to watch out for that.